src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const robotValues = Object.values(robot);
+  const robotKeys = Object.keys(robot);
+
+  if (new Set(robotValues).size !== robotValues.length) {
+    return null;
+  }
+
+  const newRobot = {};
+
+  for (let i = 0; i < robotValues.length; i++) {
```

Comment:
You can use a for...of loop with Object.entries() to loop through both keys and values at the same time, making the code more readable and efficient. 🚀